data/tutorials/gs_01_a_first_hour_with_ocaml.md

```diff
@@ -1,50 +1,62 @@
 ---
 id: first-hour
-title: A First Hour with OCaml
+title: Your First Day with OCaml
 description: >
   Discover the OCaml programming language in this longer tutorial that takes you
   from absolute beginner to someone who is able to write programs in OCaml.
 category: "getting-started"
 date: 2021-08-06T17:11:00-00:00
 ---
 
-# A First Hour with OCaml
+# Your First Day with OCaml
 
-You may follow along with this tutorial with just a basic OCaml installation,
+This tutorial will not only give you the basics of OCaml, but it will
+also guide you through creating your first applications with OCaml.
+You can follow along with a basic OCaml installation,
 as described in [Up and Running](/docs/up-and-running).
 
 Alternatively, you may follow almost all of it by running OCaml in your browser
 using the [OCaml Playground](https://ocaml.org/play), with no installation required.
 
-On macOS/iOS/iPadOS, you can download this [all-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side-by-side with an interactive toplevel. Plus, it's free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS)!
+On macOS/iOS/iPadOS, you can download this [all-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side-by-side with an interactive toplevel. Plus, it's free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS).
 
+**Note:** The OCaml code examples in this tutorial start with a prompt `#`.
+This prompt is not part of the code, it's a hint to you that the code
+following it should be either entered into an interactive toplevel, or
+into the OCaml Playground.
 ## Running OCaml Programs
 
-To try small OCaml expressions, use an interactive toplevel, or REPL
-(Read-Eval-Print Loop). The `ocaml` command provides a basic toplevel
-(install `rlwrap` through your system package manager and run `rlwrap ocaml` instead to get history navigation).
+### Using an OCaml toplevel (REPL)
 
-The alternative REPL [UTop](https://github.com/ocaml-community/utop) may be installed
-through [opam](https://opam.ocaml.org/doc/Install.html) or your system package manager. It has
-the same basic interface but is much more convenient to use (history
-navigation, auto-completion, etc.). 
+We recommend using UTop (see [here](/docs/up-and-running#using-the-ocaml-toplevel-with-utop)),
+but alternatively, the basic toplevel of OCaml (`ocaml` command) can also be used. For installation instructions,
+see [Up and Running](/docs/up-and-running#installing-ocaml).
 
-Remember you can exit either toplevel at any time with Ctrl+D or the
-built-in `exit` function: `exit 0;;`.
+If you have never used a toplevel (REPL) before, think of it as an interactive terminal/shell that
+evaluates expressions. You type an expression followed by the Enter or Return key,
```

Comment:
"type" -> "write" (type made me think of typing at first, I had to reread the sentence a few times, but maybe that's just me)

Reply:
ohh, typing in the context of types... :see_no_evil: Yes, I can see how that reads strangely.